src/lib/config.ts

```diff
@@ -39,7 +39,6 @@ down /etc/openvpn-misc/downscript.sh
 comp-lzo
 dev resin-vpn
 dev-type tun
-proto tcp
```

Comment:
We don't need this at all anymore?

Reply:
@thgreasi it's handled by the default `remote` directive [here](https://github.com/balena-io/open-balena-api/pull/1570/files#diff-0b4f5977a52cef50ed20e7cdf9761795df9ee1cb3b5e5ea45edeb9394e8a47ffR142) now..

Reply:
@Page- @thgreasi need a decision here folks. Do we add extra code and not modify the config unless a new feature switch is set or go as-is.

Reply:
@ab77 I would be happy with something like:
````
const DEVICE_CONFIG_OPENVPN_CONFIG = `
client
${
  VPN_UDP_HOST == null
    ? `remote ${VPN_HOST} ${VPN_PORT}`
    : `remote ${VPN_UDP_HOST} ${VPN_PORT} udp
remote ${VPN_HOST} ${VPN_PORT} tcp`
}
remote {{VPN_DETAILS}}
resolv-retry infinite
remote-cert-tls server
tls-version-min 1.2
ca /etc/openvpn/ca.crt
auth-user-pass /var/volatile/vpn-auth
auth-retry none
script-security 2
up /etc/openvpn-misc/upscript.sh
up-restart
down /etc/openvpn-misc/downscript.sh
comp-lzo
dev resin-vpn
dev-type tun
${VPN_UDP_HOST == null ? 'proto tcp' : ''}
nobind

persist-key
persist-tun
verb 3
user openvpn
group openvpn

reneg-bytes 0
reneg-pkts 0
reneg-sec 0
`;
```
That should give exactly the same config and we can (I can help) also add a test case for that.
PS: Not sure whether the extra space before the `nobind` would cause any issues when VPN_UDP_HOST is defined, but even in that case we could just add a `.replace('\n\nnobind', '\nnobind')`.

Reply:
Please commit your suggestion @thgreasi, or leave my version.